libr/include/r_getopt.h

```diff
@@ -11,13 +11,21 @@ Duplicated code in here:
 
 */
 
-#if __WINDOWS__
+#if 1
+// __WINDOWS__
 
 #ifndef GETOPT_C
+#if __WINDOWS__
 __declspec(dllimport) int r_optind;
 __declspec(dllimport) char *r_optarg;
 #endif
+#endif
+extern int r_optind;
```

Comment:
I think it would be better to have these variables as part of a `RGetOpt` structure and initialize them as part of `r_getopt_init`. I think we shouldn't have these global vars.

Reply:
Exactly, and it's could be easily solved by https://github.com/radareorg/radare2/issues/16322